openqa-trigger-bisect-jobs

```diff
@@ -47,28 +48,60 @@ def call(cmds, dry_run=False):
 def openqa_clone(cmds, dry_run, default_opts=['--skip-chained-deps', '--within-instance'], default_cmds=['_GROUP=0']):
     return call(['openqa-clone-job'] + default_opts + cmds + default_cmds, dry_run)
 
+def fetch_url(url, request_type='text'):
+    if url.scheme == "file":
+        decodejson = lambda c: json.loads(c)
+        path = url.geturl()
+        path = path[7:]
```

Comment:
I would avoid the magic number. I guess you want to strip the initial `file://` part? Then there are better ways to do that with an url object

Reply:
can you give an example of a better way?

Reply:
obsolete